src/types/publication.types.ts

```diff
@@ -73,7 +74,7 @@ export interface Publication {
   readonly owner: string;
   readonly status: PublicationStatus;
   entityDescription: PublicationEntityDescription;
-  fileSet: File[];
+  fileSet: any; //TODO: map fileSet correctly according to datamodel
```

Comment:
åja, skjønner de andre nå :p 
men er ikke `File` riktig da? har jo flyttet over til å matche 
```
  administrativeAgreement: boolean;
  publisherAuthority: boolean;
  embargoDate: Date | null;
  license: License | null;
```
så er det noen andre felter som kanskje bare kan kuttes, med mindre de brukes for opplasting elns

Reply:
https://github.com/BIBSYSDEV/nva-datamodel-java/blob/develop/src/main/java/no/unit/nva/model/File.java
ser ut som vi bare trenger å flytte `type -> mimeType` og `size` ut fra `data`

Reply:
ah, det skal kanskje være `publication.fileSet.files` ja?